model_external_socket/java/src/org/openda/externalsocket/SocketClient.java

```diff
@@ -0,0 +1,62 @@
+package org.openda.externalsocket;
+
+import java.io.*;
+import java.net.InetAddress;
+import java.net.Socket;
+import java.net.UnknownHostException;
+
+class SocketClient {
+
+	private final String hostAddress;
+	private int port;
+
+	SocketClient(int port) {
+		try {
+			this.hostAddress = InetAddress.getLocalHost().getHostAddress();
+		} catch (UnknownHostException e) {
+			throw new RuntimeException(e.getMessage(), e);
+		}
+		this.port = port;
+	}
+
+    String sendAndReceive(String messageIn) {
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=6157987)